helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java

```diff
@@ -523,6 +532,27 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
     }
   }
 
+  private static Map<String, String> parseListOfPairs(List<String> list, String fieldName) {
```

Comment:
The difference with argToStringMap is that it is used for Arguments that have `nargs("+")` like `--labels` in the `host` command.

I don't totally get the nuances of the nargs part in argparse4j but I modeled this one after the `--env` command, meaning the input is expected to be like

```
create -m a=1 -m b=2 -m c=3 job:version image args...
```

with `nargs("+")` here it would have just one `-m` like

```
create -m a=1 b=2 c=3 job:version image args...
```

I'm not sure if that would be a problem with all of the other arguments in JobCreateCommand though, none of them have `nargs("+")` (even though they often have multiple values like `--env` and `-p` or `--volume`) except for `args` at the end which has `nargs("*")`